tests/test_docker_images.py

```diff
@@ -0,0 +1,37 @@
+import docker
+from tests.helm_template_generator import render_chart
+import jmespath
+import pytest
+
+
+def list_docker_images():
+    extra_globals = {
+        "global.baseDomain": "foo.com",
+        "blackboxExporterEnabled": True,
+        "postgresqlEnabled": True,
+        "prometheusPostgresExporterEnabled": True,
+        "pspEnabled": True,
+        "veleroEnabled": True,
+    }
+
+    public_repo_docs = render_chart(values={"global": extra_globals})
+
+    # Listing docker images
+    search_string = "spec.template.spec.containers[*].image"
+    docker_images = []
+    for public_repo_doc in public_repo_docs:
+        docker_image_sublist = jmespath.search(search_string, public_repo_doc)
+
+        if docker_image_sublist is not None:
+            docker_images = docker_image_sublist + docker_images
+
+    return docker_images
+
+
+@pytest.mark.parametrize("docker_image", list_docker_images())
```

Comment:
I think we're going to have to mark this test as flaky. I've gotten a few 500 errors when testing:

`E           AssertionError: 'Error reading image: quay.io/astronomer/ap-houston-api:0.28.11 | Error: 500 Server Error for http+docker://localhost/v1.41/distribution/quay.io/astronomer/ap-houston-api:0.28.11/json: Internal Server Error ("Get "https://quay.io/v2/": dial tcp [2600:1f18:483:cf00:1a24:60a6:800:1de7]:443: connect: cannot assign requested address")`

```suggestion
@pytest.mark.flaky(reruns=5, reruns_delay=1)
@pytest.mark.parametrize("docker_image", list_docker_images())
```

Reply:
Looks like the above change will require adding `pytest-rerunfailures` to `requirements/chart-tests.in` and then running `make update-requirements`

Reply:
Done